packages/core/src/components/dropdown/test/filter/normalize-text-false.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Dropdown - Filter</title>
```

Comment:
Was thinking just a quick one on this since I wrote this test case before implementation of the code. 

Should it be on its own, Or can it be together with the filter folder and tests just update the path to this one where `normalized-text` is added and changed? Should be just add it with an document.addAttribute before doing the tests? How you guys feel?

Like this works but think we can do better.

Reply:
Sorry, I do not follow, can you please elaborate more?

Reply:
No problem :) Erm i'm like just questioning myself on how to organise specific tests regarding the "same" kinda of tests. So I just made another folder inside the filter folder and moved the "new" test inside of there. Think it works but please see if you think it makes sense.